api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/PaymentCycleTestDataFactory.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.dhsc.htbhf.claimant.testsupport;
+
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimTestDataFactory.aValidClaim;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aValidPaymentCycleVoucherEntitlement;
+
+public class PaymentCycleTestDataFactory {
+
+    private static final String CARD_ACCOUNT_ID = "123456789";
+    private static final int TOTAL_VOUCHERS = 1;
+    private static final int TOTAL_ENTITLEMENT_AMOUNT_IN_PENCE = 310;
+
+
+    public static PaymentCycle aValidPaymentCycle() {
+        return aValidPaymentCycleBuilder().build();
+    }
+
+    public static PaymentCycle aPaymentCycleWithClaim(Claim claim) {
+        return aValidPaymentCycleBuilder().claim(claim).build();
+    }
+
+    public static PaymentCycle aPaymentCycleWithCardAccountId(String cardAccountId) {
+        return aValidPaymentCycleBuilder().cardAccountId(cardAccountId).build();
+    }
+
+    private static PaymentCycle.PaymentCycleBuilder aValidPaymentCycleBuilder() {
```

Comment:
Minor - Would it be worthwhile having a `Payment` on the default object as its probably more realistic to have one than not.

Reply:
I would just added a method that includes it if needed, or use the builder. If you have a payment in your test you would likely want to have several other fields added as well.

Reply:
Ok, that's fine, we'll add that when we need to then.